final/evaluation.tex

```diff
@@ -180,14 +180,22 @@ \subsection{Development Methodologies}
 Naturally, there were also things that could have gone better. It is easy to
 underestimate the amount of work when putting together a sprint plan, which has
 been a source of problems. The solution for this issue is to deliberately plan
-less: it is easier to move new user stories to the sprint plan than it is to decide
-which user stories not to complete.
-
-Another issue is that the development was struck with two sources of delay.
-Spoofax changes fast, thus several times development had to stop to catch up
-with these changes. The second source of delay was the integration with the
-Eclipse IDE, which led to several unforeseen issues. Some time could have been
-saved if the client had been asked for advice sooner.\\
+less: it is easier to move new user stories to the sprint plan than it is to
+decide which user stories not to complete.
+
+A second issue concerns the unstable development environment provided by Eclipse.
+Omitting all the details, several hard to track issues occurred which took a
```

Comment:
> Why would you omit details? The whole point of my comment was to elaborate more on exactly these details.

Because these technical details to not belong in the evaluation. I suppose I could go on about how OSGi works, and linkage errors, and external dependencies, and how these surfaces as an ArrayIndexOutOfBoundsException and whatnot, but as I said: I don't think those details belong here.

Do you have any suggestions to make it less vague and less negative?